packages/nx-go/src/utils/run-go-command.helper.ts

```diff
@@ -16,10 +16,10 @@ export function runGoCommand(
 
   try {
     console.log(`Executing command: ${execute}`)
-    execSync(execute, { cwd, stdio: [0, 1, 2] })
```

Comment:
FYI it turns out that this breaks the usage of any of the go commands except `version` as the output no longer gets piped to the terminal. 

I'm patching this in #76